nomenclature/validation_schemas/region_schema.yaml

```diff
@@ -0,0 +1,34 @@
+$schema: 'https://json-schema.org/draft/2020-12/schema'
+title: RegionCodeList
+description: >
+  This schema is used for the validation of yaml region files.
+type: array
+items:
+  $ref: '#/definitions/Hierarchy'
+minItems: 1
+
+definitions:
+
+  Hierarchy:
+    type: object
+    # The key of the dictionary is the hierarchy name
```

Comment:
When experimenting with json-schema, I noticed that this doesn't work if you have a dictionary with multiple items.

If you have
```yaml
    patternProperties:
      # The key of this dictionary is the common region name
      ^.+$:
        type: array
        items:
          type: string
```
json-schema will check that one key exists that maps to a list of strings. 

In contrast, if you have
```yaml
    additionalProperties:
       type: array
       items:
          type: string
```
it will check that **all** keys map to a list of strings.

Reply:
Ah ok I think I see what you mean. I think this could be solved by adding `  "additionalProperties": false`. This way I think the schema is forced to check every entry against the patternProperty.

Reply:
Ah perfect